jina/drivers/multimodal.py

```diff
@@ -60,12 +60,22 @@ def __init__(self,
         super().__init__(traversal_paths=traversal_paths, *args, **kwargs)
 
 
+    @property
+    def position_by_modality(self) -> List[str]:
```

Comment:
The point is to have it directly as an attribute of driver and not as a property reading from executor. So that the executor does not know anything about it